pkg/csi_deployment.go

```diff
@@ -14,6 +15,7 @@ import (
 )
 
 type CSIDeployment struct {
+	ctx            context.Context
```

Comment:
I guess it's not really good practise to use context in such way. For reference  https://www.ardanlabs.com/blog/2019/09/context-package-semantics-in-go.html

Reply:
Thank you for the article. Removed context from data structs and updated receiving functions